update-framebuffer.sh

```diff
@@ -11,6 +11,9 @@ if [[ -f /etc/os-release ]]; then
   if grep Debian /etc/os-release > /dev/null; then
     apt-get install -yqq fonts-piboto
   fi
+  if getent passwd 989 > /dev/null; then
```

Comment:
maybe there is a better check to see if `lvm` and `parted` is needed.

Reply:
In this case, there will be no entry for `989`, which is why we use that UID instead of a username. On pibox-os, that UID corresponds to `kubesail-agent`, but if that user *does not exist*, we still need 989 in order to mount that directory via the KubeSail-Agent - so